src/targets/cpu/target.cpp

```diff
@@ -61,7 +61,7 @@ namespace cpu {
 
 std::string target::name() const { return "cpu"; }
 
-// cppcheck-suppress constParameter
+// cppcheck-suppress constParameterReference
```

Comment:
`get_passes` wouldn't change `gctx` AFAIK. Why suppress const-ness ?

Reply:
It gets casted to a non-const `context` which is passed to passes that take the `context` as non-const as well. Cppcheck just doesnt understand the `any_cast`.